CONTRIBUTING.md

```diff
@@ -44,10 +44,9 @@ Our process for accepting changes operates by [Pull Request (PR)](https://help.g
 
 Our branching strategy is based on [this article](https://nvie.com/posts/a-successful-git-branching-model/) which we suggest you read.
 
-+ **master** a history of releases, once merged to from develop and tagged we create a release on the play store & GitHub releases.
-+ **develop**  the actively worked on next release of the app, what we branch off of while working on new features and what we merge into upon feature completion
++ **main**  the actively worked on next release of the app, what we branch off of while working on new features and what we merge into upon feature completion
 + **feature/** or feature/\<username\>/ any branch under this directory is an actively developed feature, feature branches culminate in a PR, are merged and deleted.
- Typically a feature branch is off of develop and into develop but in rare scenarios if there is an issue in production a branch may be made off master to fix this issue, this type of feature branch must be merged to develop and master before being deleted.
+ Typically a feature branch is off of main and into develop but in rare scenarios if there is an issue in production a branch may be made off main to fix this issue, this type of feature branch must be merged to main before being deleted.
```

Comment:
This should be changed, everything is always on `main` (put the backquotes around branch names).

Reply:
> This should be changed, everything is always on the main

@kelson42, what should we change in this? I have not understood this properly. Should we remove this, as everything we make on the `main` branch?

> (put the backquotes around branch names).

i have added the backquotes around the branch names.

Reply:
Develop branch does not exist anymore, why we keep talking about it?

Reply:
Done.